From 48941765424bab4338258511bdeb3188fbdd8ca8 Mon Sep 17 00:00:00 2001
From: Jie Liu <liujie375@h-partners.com>
Date: Wed, 9 Nov 2022 16:33:11 +0800
Subject: [PATCH 1037/1646] [Backport] dma-mapping:add a new configuration for
 map_benchmark.

kunpeng inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I60AX5

Reference: https://gitee.com/openeuler/kernel/commit/145fdafddf13d37730ca0715f8afd5c981c23818

----------------------------------------------------------------------

A new configuration needs to be added for map_benchmark.

Signed-off-by: Jie Liu <liujie375@h-partners.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 8e9768c60930..5e34f1644fb0 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -6956,6 +6956,7 @@ CONFIG_DMA_REMAP=y
 CONFIG_DMA_DIRECT_REMAP=y
 CONFIG_DMA_CMA=y
 CONFIG_DMA_PERNUMA_CMA=y
+CONFIG_DMA_MAP_BENCHMARK=y
 
 #
 # Default contiguous memory area size:
-- 
2.17.1

